src/options/components/settings/index.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import styles from '../../options.css'
+
+
+const SettingsComponent = () => (
+    <div>
+        <h1 className={styles.routeTitle}>Settings</h1>
+        <p>Settings specific components to come in here.</p>
+    </div>
+)
+
+export default SettingsComponent
```

Comment:
Settings page wasn't changed. I just split it into component and container and located in correct folders